### PR TITLE
Adding documentation for zone support in vSphere

### PR DESCRIPTION
**Pull request by subramanian-neelakantan**
Writing out the Zone support documentation for VCP.

### CODE REVIEW

File: documentation/faqs.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ It has been tested on 1000 node Kubernetes cluster so far. Please refer to this
 
 
 ## Can vSphere Cloud Provider support Kubernetes Cluster spanning across multiple vCenters, ESXi Clusters and Datacenters?
-With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage.
+With Kubernetes version 1.9.x vSphere cloud provider supports Kubernetes cluster spanning across multiple ESXi clusters, vSphere Datacenters and vCenters. All Kubernetes node VMs must have access to shared storage. Also see [zone support](/vsphere-storage-for-kubernetes/documentation/zones.html) for using storage that is shared only by node VMs within a zone. Zone support does not work across multiple vCenters.
```
Comment (divyenpatel) [inline documentation/faqs.md]: Can we make declaration about zone support with another Question?
We might confuse user with this statement here, as from 1.9.0 we started supporting multi vCenter, but zone support is added in 1.14.x ...

File: documentation/zones.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+With the introduction of zones in kubernetes, the cloud provider can express the topology of the cloud infrastructure as zones. Kubernetes nodes and Volumes have labels denoting the zone information. The k8s pod scheduler is zone aware in placing pods on nodes in the required zone.
+
+vSphere Cloud Provider added Zones support since Kubernetes version 1.14.
+
+## Mark Zones and Regions in vCenter
+
+Zones and Regsions are marked by creating and applying Tags in vCenter. The Tags can be applied to a Datacenter, Cluster or ESX Host. Tags are inherited from the parent and can be overridden on the child in this hierarchy. So a Tag applied to a Datacenter is inherited by all Clusters and Hosts in that Datacenter, and a Tag applied to a Cluster overrides any Tags that it might inherit from its parent Datacenter.
```
Comment (divyenpatel) [inline documentation/zones.md]: Link to documentation on how to create tag and category would help user 

https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html

Comment (divyenpatel) [inline documentation/zones.md]: Regsions => Regions

File: documentation/zones.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
+---
+title: Zone support in vSphere Cloud Provider
+---
+
+## Introduction
+
+In the vSphere infrastructure, kubernetes nodes are run as VMs on an ESX host and Volumes are created as VMDK files on a datastore. The Volume is always placed on a datastore that is shared among all the node VMs of the k8s cluster. This is done so that the Volume can be attached to a pod scheduled on any of the node VMs. However, this implies that there should be a shared datastore among the node VMs in vSphere infrastructure to create Volumes. It also implies that the datastore capacity of any other non-shared datastores cannot be utilized for creating Volumes.
```
Comment (divyenpatel) [inline documentation/zones.md]: Can you replace "kubernetes" => "Kubernetes"  all places. also "k8s" => "Kubernetes" or "K8s"

**Comment by divyenpatel** [comment]
@subramanian-neelakantan I have provided some minor comments. Rest all looks good to me.

File: documentation/zones.md (hunk 3 above)
Comment (subramanian-neelakantan) [reply documentation/zones.md]: Done.

File: documentation/zones.md (hunk 2 above)
Comment (subramanian-neelakantan) [reply documentation/zones.md]: Done.

Comment (subramanian-neelakantan) [reply documentation/zones.md]: Good idea. Added this one.

File: documentation/faqs.md (hunk 1 above)
Comment (subramanian-neelakantan) [reply documentation/faqs.md]: Alright. I will remove this mention and leave the zone support in a separate faq.

**Comment by subramanian-neelakantan** [comment]
> @subramanian-neelakantan I have provided some minor comments. Rest all looks good to me.

I have addressed your comment in latest commit https://github.com/vmware/vsphere-storage-for-kubernetes/commit/4e7927163c7e361198e126d87d2d8a4a6508c5cd. Kindly take another look.

**Pull request merged**